client/app.jsx

```diff
@@ -63,41 +65,111 @@ const AuthorName = styled.span`
   display:inline;
   `;
 
+const TrailingInfo = styled.div`
+  color: white;
+  font-size: 14px;
+  font-family: "sf pro display", -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+  font-weight:250;
+  margin-top: 16px;
+  margin-bottom: 16px;
+  `;
+
+const SmallIcon = styled.svg`
+  width: 14px;
+  box-sizing: border-box;
+  display: inline;
+  `;
+
+const InfoText = styled.span`
+  margin-left: 8px;
+  display: inline;
+  `;
+
+const Wishlist = styled.button`
+  position: relative;
+  color: white;
+  height: 40px;
+  border: 1px solid white;
+  font-weight: 700;
+  background: black;
+  align-items: center;
+  border-radius: 4px;
+  display: inline-flex;
+  min-width: 10rem;
+  padding: 0 1.2rem;
+  `;
+
+const HeartIcon = styled.svg`
+  width: 24px;
+  height: 24px;
+  `;
+
```

Comment:
You should probably keep your styles in a separate file/folder (using 'export' before each 'const'). This way it's also easier to keep track of what you are importing and using in each React component.

Reply:
Yup -- I know I need to do this, for sure.